src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -23,7 +23,17 @@ void doNamesMatch_NoDifferentLastName() {
 
     @Test
     void doNamesMatch_NoDifferentFirstName() {
-        assertTrue(false);
```

Comment:
This is the deleted code and was only put there to force the test to fail. I changed all of the methods to return null so that I don't have to do funky stuff like assertTrue(false).